tests/FakeItEasy.Tests/Core/DefaultFakeManagerAccessorTests.cs

```diff
@@ -99,7 +99,7 @@ public void Should_fail_when_getting_manager_from_object_where_a_fake_manager_ha
             // Assert
             exception.Should()
                 .BeAnExceptionOfType<ArgumentException>()
-                .WithMessage("The specified object is not recognized as a fake object.");
+                .WithMessage($"Object '{proxy.ToString()}' of type '{proxy.GetType().FullNameCSharp()}' is not recognized as a fake object.");

```

Comment:
The problem I was getting here is that the type name was non-deterministic. Castle.Proxies.ObjectProxy_2 is sometimes named Castle.Proxies.ObjectProxy_4 or Castle.Proxies.ObjectProxy_14. The WithMessage method unfortunately doesn't allow for partial string comparisons, hence my solution above. Any other suggestion?